index.js

```diff
@@ -45,21 +45,28 @@ Airplane.prototype.land = function () {
     this.name=name;
     this.age=age;
     this.stomach=[];
-    this.eat="somefood";
+   // const someFood= stomach.push('somefood');
+   }
 //Object.prototype.toString()
-Person.prototype.toString(name,age);
-/*const array1 = [1, 2, 'a', '1a'];
-
-console.log(array1.toString());
-// expected output: "1,2,a,1a"
-*/
-};
-
-
-
-
-
-/*
+Person.prototype.toString=function(){
+  return this.name + this.age;
+}
+Person.prototype.eat=function(someFood){
+  this.stomach=[i++];
+}
+  Person.prototype.poop=function() {
+    this.stomach=[i--];
+  }
```

Comment:
These lines of code weren't necessary since you are just overwriting them with the next block of code